### ISSUE TITLE
CI: Running jest results into warning and exits the process

### ISSUE THREAD

**Issue by kettanaito** [kettanaito/atomic-layout#114]
## What:

When performing a build on CircleCI, and running unit tests with Jest, the following message is output:

```
ts-jest[versions] (WARN) Version 24.0.0-alpha.9 of jest installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=22.0.0 <24.0.0). Please do not report issues in ts-jest if you are using unsupported versions.
```

> This behavior does occur *sometimes*, but there are no passing tests output into the terminal of the CI anyway

## Why:

Used version of Jest is not tested with `ts-jest`.

## Environment:

- atomic-layout version: `0.6.2`

## Expected behavior:

* Unit tests run and perform properly
* The only way a test may exit a process is when it fails

**Comment by kettanaito** [comment]
Should be fixed by 3134dc3, performing testing.

**Comment by kettanaito** [comment]
CI passed, the issue is resolved by reverting to `jest@23`.
